components/button-cards/generic-card/generic-card.scss

```diff
@@ -125,7 +125,7 @@
       fill: $clear-blue;
     }
 
-    .mdsds-generic-card__body-text-above-headline {
+    .msds-generic-card__body-text-above-headline {
```

Comment:
I think the text-above headline should be 

      @extend .msds-text-header-5;

Check out the design - https://projects.invisionapp.com/d/main?origin=v7#/console/14470201/361186121/inspect?scrollOffset=249.3333282470703

Reply:
It is `@extend .msds-text-header-5;`